numpy/core/fromnumeric.py

```diff
@@ -886,7 +887,7 @@ def sort(a, axis=-1, kind=None, order=None):
     =========== ======= ============= ============ ========
        kind      speed   worst case    work space   stable
     =========== ======= ============= ============ ========
-    'quicksort'    1     O(n^2)            0          no
+    'quicksort'    1     O(n*log(n))       0          no
```

Comment:
I don't think we can call this 'quicksort' and show `O(n*log(n))`, people will just be confused. I suggest making a table for for all the different sort types, adding `introsort`, `timsort`, and `radixsort`, then an explanation of what algorithms are used for the different keyword arguments. We might also consider using some more aliases for the keywords.